packages/pasteup/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@guardian/pasteup",
-  "version": "1.0.0-alpha.11",
+  "version": "2.0.0-alpha.0",
```

Comment:
As we're alpha, we can probably keep it on 1. Our versioning seems a bit weird tbh - we should be pre-version-1 if alpha really.

Reply:
My concern would be that someone didn’t heed the warning and has used the npm module and a minor update would then break their code. However, I think I can instead use [pre release versioning](https://docs.npmjs.com/misc/semver#prerelease-tags) but need to release properly. Will read up!

Reply:
⍺⁴𝙻